source_gen/lib/builder.dart

```diff
@@ -82,7 +82,10 @@ class CombiningBuilder implements Builder {
   @override
   Future<void> build(BuildStep buildStep) async {
     // Pattern used for `findAssets`, which must be glob-compatible
-    final pattern = buildStep.inputId.changeExtension('.*$_partFiles').path;
+    final pattern = buildStep.inputId
```

Comment:
I think we should probably do `Glob.quote` here instead? That will handle other characters as well.

Reply:
Thanks for your help! I didn't know it existed.